content/docs/releases/README.md

```diff
@@ -19,16 +19,17 @@ should be stable enough to run.
 <a id="supported-releases"></a>
 ## Currently supported releases
 
-| Release      | Release Date | End of Life            | [Supported Kubernetes / OpenShift Versions][s] | [Tested Kubernetes Versions][test] |
-|:------------:|:------------:|:----------------------:|:----------------------------------------------:|:----------------------------------:|
-| [1.17][]     | Feb 03, 2025 | Release of 1.19        |       1.29 → 1.32   /   4.16 → 4.17            |  1.29 → 1.32                       |
-| [1.16][]     | Oct 03, 2024 | Release of 1.18        |       1.25 → 1.32   /   4.14 → 4.17            |  1.27 → 1.31                       |
+| Release  | Release Date | End of Life     | [Supported Kubernetes / OpenShift Versions][s] | [Tested Kubernetes Versions][test] |
+|:--------:|:------------:|:---------------:|:----------------------------------------------:|:----------------------------------:|
+| [1.18][] | Jun 10, 2025 | Release of 1.20 | 1.30 → 1.33   /   4.16 → 4.18                  | 1.30 → 1.33                        |
+| [1.17][] | Feb 03, 2025 | Release of 1.19 | 1.29 → 1.32   /   4.16 → 4.18                  | 1.29 → 1.32                        |
```

Comment:
We're currently using Kind 0.27.0 which has node images for K8S 1.29 - 1.33
 * https://github.com/kubernetes-sigs/kind/releases/tag/v0.27.0

The latest version does not have prebuilt node images for 1.29, so we may have to change this page when we upgrade Kind:
 * https://github.com/kubernetes-sigs/kind/releases/tag/v0.29.0

Reply:
Is there a reason we don't support K8s 1.33 with cert-manager 1.17? I think it should work well, and is also tested (I think).

Reply:
We support it, but the testing config hasn't updated.
I've updated the "tested versions" column to  reflect the current situation.